I'm sorry, but an uncaught exception occurred.

After initialization, but before game start.
  File "game/screens.rpy", line 546, in prepare_screen
    screen about():
  File "game/screens.rpy", line 546, in prepare
    screen about():
  File "game/screens.rpy", line 553, in prepare
    use game_menu(_("About"), scroll="viewport"):
  File "game/screens.rpy", line 553, in prepare
    use game_menu(_("About"), scroll="viewport"):
  File "game/screens.rpy", line 557, in prepare
    vbox:
  File "game/screens.rpy", line 557, in prepare
    vbox:
  File "game/screens.rpy", line 566, in prepare
    text _("Made with {a=https://www.renpy.org/}Ren'Py{/a} [renpy.version_only].\n\n[renpy.license!t]" \n)
SyntaxError: unexpected character after line continuation character (screens.rpy, line 566)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/main.py", line 662, in main
    run(restart)
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/main.py", line 97, in run
    renpy.display.screen.prepare_screens()
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/display/screen.py", line 955, in prepare_screens
    s.ast.prepare_screen()
  File "game/screens.rpy", line 546, in prepare_screen
    screen about():
  File "game/screens.rpy", line 546, in prepare
    screen about():
  File "game/screens.rpy", line 553, in prepare
    use game_menu(_("About"), scroll="viewport"):
  File "game/screens.rpy", line 553, in prepare
    use game_menu(_("About"), scroll="viewport"):
  File "game/screens.rpy", line 557, in prepare
    vbox:
  File "game/screens.rpy", line 557, in prepare
    vbox:
  File "game/screens.rpy", line 566, in prepare
    text _("Made with {a=https://www.renpy.org/}Ren'Py{/a} [renpy.version_only].\n\n[renpy.license!t]" \n)
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/pyanalysis.py", line 714, in ast_eval
    return self.ast_eval_literal(expr)[0]
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/pyanalysis.py", line 696, in ast_eval_literal
    expr = py_compile(expr, 'eval', ast_node=True)
  File "C:\Users\djkni\Downloads\renpy-7.4.11-sdk/renpy/python.py", line 835, in py_compile
    raise e
SyntaxError: unexpected character after line continuation character (screens.rpy, line 566)

Windows-10-10.0.22000
Ren'Py 7.4.11.2266
Echoes That Sing 1.0
Mon Mar 28 18:39:09 2022
